### PR TITLE
Someone doesn't understand how Futures work. Hint, it's me

### PR DESCRIPTION
**Pull request by geoffjentry**
@mcovarr @cjllanwarne - given that you all are hip to the topic, can you take a look and see if looks right?

### CODE REVIEW

File: engine/src/main/scala/cromwell/engine/backend/jes/JesBackend.scala [hunk 1]
```diff
@@ -141,15 +141,15 @@ object JesBackend {
     val delay = backoff.backoffMillis.millis
 
     if (retries > 0) {
-      f recoverWith {
+      f() recoverWith {
```
Comment (geoffjentry) [inline engine/src/main/scala/cromwell/engine/backend/jes/JesBackend.scala]: I suppose I could pull the `f()` into a single spot. I'll leave it up to reviewers.

Comment (mcovarr) [inline engine/src/main/scala/cromwell/engine/backend/jes/JesBackend.scala]: I think it's fine, particularly since I did it this way on develop.  😛

**Comment by mcovarr** [comment]
:+1: assuming Travis approves

**Comment by geoffjentry** [comment]
@mcovarr Can't assume anything like that these days, at least when it comes to me.

**Comment by cjllanwarne** [comment]
👍

**Pull request merged**